js/script.js

```diff
@@ -0,0 +1,61 @@
+import { toggleMenu } from "./utils/menu-button.js";
+import { currentTime } from "./utils/current-time.js";
+import { calculatorApp } from "./utils/calculator.js"
+import { createModal, deleteModal} from "./dom-utils/modal.js";
+import { createCalc } from "./dom-utils/createCalc.js";
+import { toggleCalc } from './utils/calc-app.js';
+import { toggleMedia } from './utils/media-app.js'
+import { toggleGallery } from './utils/media-toggle.js'
+import { popUp } from './utils/gallery-popup.js';
+
+const menuButton = document.getElementById('menuButton');
+const calculator = document.getElementById('calculator');
+const media = document.getElementById('mediaPlayer');
+const myPictures = document.getElementById('myPictures');
+const closeButtonCalc = document.getElementById('closeCalc');
+const closeButtonMedia = document.getElementById('closeMedia');
+const closeButtonGallery = document.getElementById('closeGallery');
+
+menuButton.addEventListener("click", (e) => {
+    e.preventDefault();
+    toggleMenu();
+})
+
+calculator.addEventListener("click", (e) => {
+    e.preventDefault();
+    // createModal();
```

Comment:
const modal = createModal() -> return the modal div
onst calculator = creaetCalculator()
modal.innerHTML(calculator)

Reply:
parent.appendChild(modal)